packages/hoprd/crates/hoprd-misc/src/cli.rs

```diff
@@ -27,10 +31,9 @@ pub const DEFAULT_HEALTH_CHECK_PORT: u16 = 8080;
 
 pub const MINIMAL_API_TOKEN_LENGTH: usize = 8;
 
-regex!(is_ipv4_host "^[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}[:]{1}[0-9]{1,6}$");
-
 fn parse_host(s: &str) -> Result<crate::config::Host, String> {
-    if !is_ipv4_host(s) {
+    let host = s.split_once(":").map_or(s, |(h, _)| h);
+    if !(validator::validate_ip_v4(host) || is_dns_address(host)) {
```

Comment:
I'd switch the check name to `is_domain`

Reply:
As agreed will be done in config PR.